config/300-cloudbuildsource.yaml

```diff
@@ -0,0 +1,132 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+apiVersion: apiextensions.k8s.io/v1beta1
+kind: CustomResourceDefinition
+metadata:
+  labels:
+    duck.knative.dev/source: "true"
+    events.cloud.google.com/release: devel
+    events.cloud.google.com/crd-install: "true"
+  annotations:
+    registry.knative.dev/eventTypes: |
+      [
+        { "type": "com.google.cloud.build.event", "description": "This event is sent when your build's state changes such as when your build is created, when your build transitions to a working state, and when your build completes."}
```

Comment:
I recall you mentioned you receive two notifications... when the build starts and when it ends. Was wondering if we can split this into two events for finer grain filtering...
I think you are parsing the message in the converter, is that the case? Can we read from some property what type of event was and populate the eventType with the proper value...

Reply:
We will receive event when build's state changes, such as when build is created, when build transitions to a working state, and when build completes. FYI: https://cloud.google.com/cloud-build/docs/api/reference/rest/Shared.Types/Status
I already embed the status info into the subject of event

Reply:
synced offline... We are leaving it as is for now...

nit: can you add a `,` before `such as` in the description

Reply:
We leave the schema for a follow up PR? I'm fine with that